server/database/models/lunch.js

```diff
@@ -0,0 +1,18 @@
+const Sequelize = require('sequelize');
+const sequelize = require('./sequelize.js');
+
+const Lunch = sequelize.define('lunch', {
+  price: {
+    type: Sequelize.FLOAT,
+  },
+  note: {
```

Comment:
It would be better if you make the note ```TEXT ``` instead of ```string```

Reply:
done :+1: